client/my-sites/controller.js

```diff
@@ -271,9 +265,6 @@ module.exports = {
 		 * Sites is rendered on #primary but it doesn't expect a sidebar to exist
 		 * so section needs to be set explicitly and #secondary cleaned up
 		 */
-		context.store.dispatch( uiActions.setSection( 'sites', {
```

Comment:
Yes, we should move this selector out of primary or force it to go through `/sites`.